### PR TITLE
Exclude the var directory from the backup

### PR DESCRIPTION
**Pull request by lzammit**
# Description

Exclude the var directory from the backup
# Impacts

database-backup-and-maintenance.sh script and backups
# Delete branch after merge

YES

### CODE REVIEW

File: addons/database-backup-and-maintenance.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ fi
 # Backup complete PacketFence installation except logs
 current_tgz=$BACKUP_DIRECTORY/$BACKUP_PF_FILENAME-`date +%F_%Hh%M`.tgz
 if [ ! -f $BACKUP_DIRECTORY$BACKUP_PF_FILENAME ]; then
-    tar -czf $current_tgz $PF_DIRECTORY --exclude=$PF_DIRECTORY_EXCLUDED
+    tar -czf $current_tgz $PF_DIRECTORY --exclude=$PF_DIRECTORY_EXCLUDED --exclude=$PF_DIRECTORY\var
```
Comment (whitx) [inline addons/database-backup-and-maintenance.sh]: / instead of \

**Comment by fdurand** [comment]
PF_DIRECTORY_EXCLUDED is not use anymore

File: addons/database-backup-and-maintenance.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ fi
 # Backup complete PacketFence installation except logs
 current_tgz=$BACKUP_DIRECTORY/$BACKUP_PF_FILENAME-`date +%F_%Hh%M`.tgz
 if [ ! -f $BACKUP_DIRECTORY$BACKUP_PF_FILENAME ]; then
-    tar -czf $current_tgz $PF_DIRECTORY --exclude=$PF_DIRECTORY_EXCLUDED --exclude=$PF_DIRECTORY\var
+    tar -czf $current_tgz $PF_DIRECTORY --exclude=$PF_DIRECTORY'logs/*' --exclude=$PF_DIRECTORY'var/*'
```
Comment (whitx) [inline addons/database-backup-and-maintenance.sh]: $PF_DIRECTORY_EXCLUDED was logs directory already tho

**Comment by tlaurion** [comment]
Ok. Merging!

**Pull request merged**